Library/LocalizedString.swift

```diff
@@ -1,5 +1,18 @@
 import Prelude
 
+public func localizedPostalCode() -> String {
+  // 🙏 Inspired by Kristina Fox
+  // https://speakerdeck.com/krstnfx/internationalizing-your-app?slide=37
+  switch AppEnvironment.current.locale.regionCode {
+  // 🇺🇸
+  case .some("US"): return localizedString(key: "Zip_code", defaultValue: "Zip code")
+  // 🇨🇦
+  case .some("CA"): return localizedString(key: "Postal_code", defaultValue: "Postal code")
+  // 🌍
+  default: return localizedString(key: "Post_code", defaultValue: "Post code")
```

Comment:
I think _Post code_ should be one word (_postcode_). I've rarely seen it as two words in the wild (in Australia). Big fan of this PR otherwise.